R/combinedscatterplot.R

```diff
@@ -73,6 +73,14 @@
 #' @param quadrant.bottom.right.title.font.family Font family of the bottom right quadrant title
 #' @param quadrant.bottom.right.title.font.size Font size of the bottom right quadrant title
 #' @param quadrant.bottom.right.title.font.color Font color of the bottom right quadrant title
+#' @param plot.border.show Boolean toggle to show border around plot area. 
+#'  If this is true, then \code{plot.border.*} overrides \code{x.line.*} and \code{y.line.*}.
+#' @param plot.border.color Color of border around plot area (Default is black).
+#' @param plot.border.width Width of border around plot area in px (Default is 1).
+#' @param fixed.aspect Whether of not to force the x and y axis to be at the same scale. Default to FALSE. 
+#'  Cannot be guarenteed if any of the axis bounds are set.
+#' @param origin Defaults to FALSE. Shows the origin lines as dotted if not along axis.
```

Comment:
Do we need this? I assume it toggles the zero-line? Is it always dotted?

Reply:
Thanks for picking this up. I've removed it in [161b4ae](https://github.com/Displayr/flipStandardCharts/pull/97/commits/161b4ae7dc3eac97b399ec79bdf11dfee2788841). If `origin` is false then it turns off the zero lines, but since by default `x.zero.line.width = 0` and `y.zero.line.width = 0` there's no point keeping it.